staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -264,39 +351,20 @@ func (o *AuditWebhookOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.ConfigFile, "audit-webhook-config-file", o.ConfigFile,
 		"Path to a kubeconfig formatted file that defines the audit webhook configuration."+
 			" Requires the 'AdvancedAuditing' feature gate.")
-	fs.StringVar(&o.Mode, "audit-webhook-mode", o.Mode,
-		"Strategy for sending audit events. Blocking indicates sending events should block"+
-			" server responses. Batch causes the webhook to buffer and send events"+
-			" asynchronously. Known modes are "+strings.Join(pluginwebhook.AllowedModes, ",")+".")
-	fs.IntVar(&o.BatchConfig.BufferSize, "audit-webhook-batch-buffer-size",
-		o.BatchConfig.BufferSize, "The size of the buffer to store events before "+
-			"batching and sending to the webhook. Only used in batch mode.")
-	fs.IntVar(&o.BatchConfig.MaxBatchSize, "audit-webhook-batch-max-size",
-		o.BatchConfig.MaxBatchSize, "The maximum size of a batch sent to the webhook. "+
-			"Only used in batch mode.")
-	fs.DurationVar(&o.BatchConfig.MaxBatchWait, "audit-webhook-batch-max-wait",
-		o.BatchConfig.MaxBatchWait, "The amount of time to wait before force sending the "+
-			"batch that hadn't reached the max size. Only used in batch mode.")
-	fs.Float32Var(&o.BatchConfig.ThrottleQPS, "audit-webhook-batch-throttle-qps",
-		o.BatchConfig.ThrottleQPS, "Maximum average number of requests per second. "+
-			"Only used in batch mode.")
-	fs.IntVar(&o.BatchConfig.ThrottleBurst, "audit-webhook-batch-throttle-burst",
-		o.BatchConfig.ThrottleBurst, "Maximum number of requests sent at the same "+
-			"moment if ThrottleQPS was not utilized before. Only used in batch mode.")
-	fs.DurationVar(&o.BatchConfig.InitialBackoff, "audit-webhook-batch-initial-backoff",
-		o.BatchConfig.InitialBackoff, "The amount of time to wait before retrying the "+
-			"first failed requests. Only used in batch mode.")
+	fs.DurationVar(&o.InitialBackoff, "audit-webhook-initial-backoff",
```

Comment:
Unfortunately you can't just change flags (see [deprecation policy, #5b](https://kubernetes.io/docs/reference/deprecation-policy/#deprecating-a-flag-or-cli)). I suggest marking the old one as deprecated, and making it an alias for the new one.